docs/modules/publisher-migrationpath.md

```diff
@@ -17,63 +17,43 @@
-For this set of methods, the encoding is JSON and no compression or payload chunking mechanism is applied in order to ensure the backwards compatibility with the 2.5.x version of OPC Publisher module.
+**Limitations:** Continuation points for GetConfiguredEndpoints and GetConfiguredNodesOnEndpoint aren't available in 2.8.2
 
-**Limitations:**
- - Continuation points for GetConfiguredEndpoints and GetConfiguredNodesOnEndpoint are not available in 2.8.2
+## OPC Publisher 2.5.x direct methods supported in 2.8.2
 
-### Direct Methods of version 2.5.x
+The following table describes the direct methods, which were available in OPC Publisher 2.5.x with their request and response payloads.
 
-The following  table describes the direct methods which were available in OPC Publisher 2.5.x with request and response.
+| **MethodName**                   | **Request**                                                     | **Response**                                                                                                                                                                                   | **in 2.8.2 and above** |
+| -------------------------------- | --------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------ |
+| **PublishNodes**                 | EndpointUrl, List\<OpcNodes\>,  UseSecurity, UserName, Password | Status, List\<StatusResponse\>                                                                                                                                                                 | Yes          |
+| **UnpublishNodes**               | EndpointUrl, List\<OpcNodes\>                                   | Status, List\<StatusResponse\>                                                                                                                                                                 | Yes          |
+| **UnpublishAllNodes**            | EndpointUrl                                                     | Status, List\<StatusResponse\>                                                                                                                                                                 | Yes          |
+| **GetConfiguredEndpoints**       | -                                                               | List\<EndpointUrl\>                                                                                                                                                                            | Yes          |
+| **GetConfiguredNodesOnEndpoint** | EndpointUrl                                                     | EndpointUrl, List< OpcNodeOnEndpointModel > where OpcNodeOnEndpointModel contains: Id ExpandedNodeId OpcSamplingInterval OpcPublishingInterval DisplayName HeartbeatInterval SkipFirst | Yes          |
```

Comment:
>SkipFirst [](http://example.com/codeflow?start=276&length=9)

Is this still supported?

Reply:
This part talks about direct methods of `2.5` version of OPC Publisher. `SkipFirst` is supported there.